### DISCUSSION TITLE
调用直链如何防止被盗刷滥用？

### DISCUSSION THREAD

**Discussion by yexiaoyu123** [AlistGo/alist#2080] [Q&A]
如题 
直接使用 www.example.com/d/xxx.zip 的 直链 老被盗刷
找了下文档没发现有防盗相关的设置.

**Comment by 12189108** [comment]
https://alist.nn.ci/config/global.html#link-expiration

**Reply by yexiaoyu123** [reply]
大佬 这个sign怎么生成？计算方式是什么？

**Reply by 12189108** [reply]
https://github.com/alist-org/alist/tree/02df3759df423f7c9a9d82133df9e328c6c6e179/pkg/sign
大概在这，没学过go

**Comment by anwen-anyi** [comment]
加密码，然设置有效期多久即可~嘿嘿，sign是怎么计算生成的就得问开发者了 哈哈。。。
